build.gradle.kts

```diff
@@ -32,8 +32,7 @@ plugins {
   // https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html
   // https://github.com/JetBrains/intellij-platform-gradle-plugin/releases
   // https://plugins.gradle.org/plugin/org.jetbrains.kotlin.jvm
-  // TODO(jwren) "2.2.0" can't be used to build 2023.3, as soon as we don't support this version, update the version here and elsewhere:
-  id("org.jetbrains.intellij.platform") version "2.4.0"
+  id("org.jetbrains.intellij.platform") version "2.5.0"
```

Comment:
Aside: would it make sense to store this in the product matrix file to keep related configuration in one place?

Reply:
If we have too I suppose, I sure hope that we don't have to have different settings here for each version shipped.  As you see in the comment, I have been predicating our support of older versions on us not fragmenting the number of inputs even more.